src/poker.js

```diff
@@ -1,5 +1,128 @@
 class Poker {
-  // Write your implementation here
+  //convert the letters (J, Q, K, A) to numbers and make all strings to numbers
+  // When there are multiple arrays in an array, combine it into one array
+  lettersToNumbers(cards) {
+    const cardsCombined = cards.flat();
+    const cardsInNumbers = [];
+    for (let card of cardsCombined) {
+      if (Number(card)) {
+        cardsInNumbers.push(Number(card));
+      } else {
+        switch (card) {
+          case "J":
+            cardsInNumbers.push(11);
+            break;
+          case "Q":
+            cardsInNumbers.push(12);
+            break;
+          case "K":
+            cardsInNumbers.push(13);
+            break;
+          case "A":
+            cardsInNumbers.push(14);
+            break;
+        }
+      }
+    }
+    return cardsInNumbers;
+  }
```

Comment:
perhaps it would be more simple to make an object, like in the scrabble challenge exemplar

Reply:
Totally...!